manifests/client.pp

```diff
@@ -120,6 +137,10 @@
     }
   }
 
+  package {
```

Comment:
I wonder if we should include stdlib and use ensure_packages here...

Reply:
I could, but how high is the chance that someone installs xymon over another way than this module? And if someone does he/she most probably wouldn't use this module then...

Reply:
Agreed